common/src/services/user.service.ts

```diff
@@ -156,6 +156,11 @@ export class UserService implements UserServiceAbstraction {
         return false;
     }
 
+    async canManageSponsorships(): Promise<boolean> {
+        const orgs = await this.getAllOrganizations();
+        return orgs.some(o => o.familySponsorshipAvailable || o.familySponsorshipFriendlyName !== null);
+    }
```

Comment:
User service doesn't feel like the place for this code. Probably ok since the Account Switching PR moves this to an org service.

Reply:
Yeah, dw about it, I'll move this later